x-vector/src/X/Data/Vector/Generic.hs

```diff
@@ -14,12 +22,19 @@ module X.Data.Vector.Generic (
   , mapMaybeM
   , imapMaybeM
 
+  -- * Modifying vectors
+
+  -- ** Transposition
+  , transpose
+
```

Comment:
There isn't, I'm not sure what it would be as there isn't an obvious canonical binary representation for vectors.